package.json

```diff
@@ -51,6 +55,8 @@
     "eslint": "^7.8.1",
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.4",
+    "fp-ts": "^2.8.2",
```

Comment:
Let's not rely on specific third-party libraries for the tests and instead boil the test case down to a unit test.